evm/src/cpu/kernel/asm/curve/bls381/util.asm

```diff
@@ -44,10 +44,80 @@ global test_add_fp381:
     %add_fp381
     %jump(0xdeadbeef)
 
+global test_sub_fp381:
+    %sub_fp381
+    %jump(0xdeadbeef)
+
 global test_mul_fp381:
     %mul_fp381
     %jump(0xdeadbeef)
```

Comment:
This shouldn't be in the kernel, because it's dead code in production. If you want to do tests this way, you can add a conditional compilation feature to the assembler—this shouldn't be difficult.

Reply:
I can just remove these tests for now, since they do work / subsequently more complicated functions which call them pass tests as well.